tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +115,26 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
-
-@pytest.mark.skip
+    assert result == tai.swap_best_by_category(
+        other=jesse,
+        my_priority="Clothing",
+        their_priority="Decor"
+    )
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_c in tai.inventory
+    assert item_f in jesse.inventory
+    assert item_f not in tai.inventory
+    assert item_c not in jesse.inventory
```

Comment:
I recommend checking all of the items here to make sure that the exact items we expect are in each inventory:  For example:
```
    assert item_c in jesse.inventory and item_d in jesse.inventory and item_e in jesse.inventory
    assert item_f in tai.inventory and item_a in tai.inventory and item_b in tai.inventory
```